source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+This feature currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
+How Your Data is Used
+---------------------
+
+When you query with natural language in Compass, the schema of the 
+collection you are querying including the collection name, field names, 
+and types. 
```

Comment:
(something's missing)
I think we're missing a verb here? Is it "... collection you are querying INCLUDES the collection ..." 

Or:
The schema ... is shared with the Gen AI provider.

Reply:
This sounds right to me as is, this is suggesting the schema, the collection name field name and data types. I think the schema is this regard is referring to the data model, such as embedded documents that could influence query results. I.E. a find query vs a flat document vs a find query of embedded arrays etc.